scipy/fft/tests/test_fftlog.py

```diff
@@ -1,13 +1,20 @@
 import warnings
 import numpy as np
-from numpy.testing import assert_allclose
 import pytest
 
 from scipy.fft._fftlog import fht, ifht, fhtoffset
 from scipy.special import poch
+from scipy.conftest import (
+    array_api_compatible,
+    skip_if_array_api_backend
+)
+from scipy._lib._array_api import xp_assert_close
 
 
-def test_fht_agrees_with_fftlog():
+# https://github.com/pytorch/pytorch/issues/59786
+@skip_if_array_api_backend('torch')
```

Comment:
Could we use `xp.flip` instead of slicing?

Reply:
Should be `A = xp.flip(A, axis=-1)`

Reply:
Addressed https://github.com/scipy/scipy/pull/19005/commits/4eef6be779a71102cbde48add42584030c163428.